lib/carbon/instrumentation.py

```diff
@@ -116,9 +116,9 @@ def recordMetrics():
   # aggregator metrics
   elif settings.program == 'carbon-aggregator':
     record = aggregator_record
-    record('allocatedBuffers', len(BufferManager))
+    record('allocatedBuffers', len(carbon.aggregator.buffers.BufferManager))
```

Comment:
I suspect this won't work. Instead, I'll likely have to rework how this stat is instrumented.